### ISSUE TITLE
proto/: No such file or directory when run `make` under tools-v2 package

### ISSUE THREAD

**Issue by Sindweller** [opencurve/curve#2057]
**Describe the bug (描述bug)**
proto/: No such file or directory

may because of we haven't left `proto/` dir under `tools-v2`

**To Reproduce (复现方法)**

`cd` to `~/curve/tools-v2` and run

```
sindweller@xindeweiladeMacBook-Pro tools-v2 % make
proto/: No such file or directory
```

**Expected behavior (期望行为)**

`make` success

**Versions (各种版本)**
OS: 
Compiler:
branch:
commit id:

**Additional context/screenshots (更多上下文/截图)**

So maybe we can add `mkdir -p proto` in `mk-proto.sh` as the first command, since we ignored `/tools-v2/proto` in `.gitignore` ? (`-p` to avoid confict if there's already a `proto/`)

like

```
# proto
mkdir -p proto
protoc --go_out=proto --proto_path=internal/proto \
    internal/proto/curvebs/topology/statuscode.proto
## curvebs
...
```

If you agree with my idea above, I can provide a pr for it, thanks.

**Comment by Cyber-SiKu** [comment]
This will be fixed in this [pr#2055](https://github.com/opencurve/curve/pull/2055).

**Comment by Cyber-SiKu** [comment]
Thanks for your work!

**Comment by Sindweller** [comment]
> This will be fixed in this [pr#2055](https://github.com/opencurve/curve/pull/2055). Great minds think alike.

great！looking forward to merge soon. I'll close this issue.
